app/src/main/aidl/net/programmierecke/radiodroid2/IPlayerService.aidl

```diff
@@ -10,6 +10,8 @@ void Play(boolean isAlarm);
 void Pause();
 void Resume();
 void Stop();
+void Next();
+void Previous();
```

Comment:
`SkipToNext`, `SkipToPrevious` to be consistent with later code.

Reply:
OK